index.html

```diff
@@ -0,0 +1,321 @@
+        <div class="snapt">
+          <img src="./images/snapshot4.png" alt="snapshot" class="responsive"/>
+        </div>
+        <div class="titol">
+          <div class="w-title">
+            <h2 class="name">Multi-Post Stories</h2>
+            <ul class="canop">
+              <li>FACEBOOK</li>
+              <li>Full Stack Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <div class="p-text">
+            <p class="word">
+              Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
+            </p>
+          </div>
+          <div>
+            <ul class="languages-2">
+              <li>html</li>
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+          </div>
+          <img src="./images/see-project.png" alt="button" class="blind">
+        </div>
+      </div>
+      <div class="card1">
+        <div class="snapshooti">
+          <img src="./images/snapshot.png" alt="snapshot" class="responsive" />
+        </div>
+        <div class="word-title">
+          <div class="w-title">
+            <h2 class="name">Facebook 360</h2>
+            <ul class="canop">
+              <li>FACEBOOK</li>
+              <li>Full Stack Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <div class="p-text">
+            <p class="word">
+              Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.
+            </p>
+          </div>
+          <div>
+            <ul class="languages">
+              <li>html</li>
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+          </div>
+          <img src="./images/Hover.png" alt="button" class="see">
+        </div>
+      </div>
+      <div class="card2">
+        <div class="snapt">
+          <img src="./images/snapshot2.png" alt="snapshot" class="responsive">
+        </div>
+        <div class="titol">
+          <div class="w-title">
+            <h2 class="name">Uber Navigation</h2>
+            <ul class="canop">
+              <li>Uber</li>
+              <li>Lead developer</li>
+              <li>2018</li>
+            </ul>
+          </div>
+          <div class="p-text">
+            <p class="word">
+              A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.
+            </p>
+          </div>
+          <div>
+            <ul class="languages-2">
+              <li>html</li>
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+          </div>
+          <img src="./images/see-project.png" alt="button" class="blind">
+        </div>
+      </div>
```

Comment:
- [ ] You have used an image in place of a `<button>` tag for the "See Project" button. Please remove the image and use a button tag instead. Do this for all your buttons.
- [ ] Also kindly implement the button default, hover, and active states as shown in the Figma design in my earlier suggestions ☝️
![image](https://user-images.githubusercontent.com/46839548/166105675-ca7e6c0d-54bf-4cb3-ad34-5f073ab046d1.png)